qcodes/utils/dataset/doNd.py

```diff
@@ -501,6 +501,14 @@ def num_points(self) -> int:
         """
         pass
 
+    @property
+    @abstractmethod
+    def action(self) -> ActionsT:
```

Comment:
perhaps we could be more descriptive? e.g. something like `post_actions` ?

Reply:
agreed